configs/azure_iot_build_rules.cmake

```diff
@@ -168,7 +168,27 @@ macro(compileAsC11)
   endif()
 endmacro(compileAsC11)
 
-compileAsC99()
```

Comment:
Please leave for backward compatibility

Reply:
talked offline.

---
In reply to: [232019614](https://github.com/Azure/azure-c-shared-utility/pull/223#discussion_r232019614) [](ancestors = 232019614)